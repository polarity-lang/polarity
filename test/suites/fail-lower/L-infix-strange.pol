infix _ + _ + _ := Fun(_,_)