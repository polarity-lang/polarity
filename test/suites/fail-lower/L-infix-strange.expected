L-019

  × Invalid infix declaration: More than one operator on left hand side
   ╭─[L-infix-strange.pol:1:1]
 1 │ infix _ + _ + _ := Fun(_,_)
   · ───────────────────────────
   ╰────
